client/state/sites/terms/selectors.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * External dependencies
+ */
+import get from 'lodash/get';
+
+/**
+ * Returns term IDs for a site, filtered by taxonomy.
+ *
+ * @param  {Object} state    Global state tree
+ * @param  {Number} siteId   Site ID
+ * @param  {String} taxonomy Taxonomy slug
+ * @return {Array}           Terms
+ */
+export function getSiteTaxonomyTermIds( state, siteId, taxonomy ) {
+	return get( state.site.taxonomies, [ siteId, taxonomy ], [] );
```

Comment:
These selectors were remnants from my mis-step of using them in the reducer.  I'll nuke and pave here.